handler/core/handler/common/customTask.conf

```diff
@@ -0,0 +1,25 @@
+tasks {
```

Comment:
Do we need those files. I think it is the IDEA problem that it copies those files to the project root folder during tests execution (especially when those test fail).